bundles/org.openhab.binding.wlanthermo/src/main/java/org/openhab/binding/wlanthermo/internal/api/mini/WlanThermoMiniCommandHandler.java

```diff
@@ -33,7 +36,21 @@
 
     public State getState(ChannelUID channelUID, App app) {
         State state = null;
-        if ("system".equals(channelUID.getGroupId())) {
+
+        String groupId = channelUID.getGroupId();
+        if (groupId == null || app == null) {
+            return null;
+        }
+
+        Unit<Temperature> unit;
+        if (app.getTempUnit().equals("fahrenheit")) {
```

Comment:
So the mini uses "fahrenheit" instead of "F"?

Reply:
yes, the API implementations unfortunately differ between the devices.